packages/allow-scripts/Containerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:16-alpine
+USER root
+COPY . /app
+# can be removed when compatible allow-scripts has been released
+RUN yarn global add file:///app \
+  && rm -r /app
```

Comment:
What's this for?

Reply:
Not intended for merge, was just a hacky way to get the updated package installed without switching registry. Will remove.